src/utils/__tests__/getPriceGuidance.test.ts

```diff
@@ -10,6 +14,33 @@ describe("#getPriceGuidance", () => {
   afterEach(() => {
     mockMetaphysics.mockClear()
   })
+
+  it("requests the five cheapest items in the given collection that still have a price", () => {
```

Comment:
Note that we discussed while pairing that this snapshot test is probably the most effective way for us to verify that we are getting the correct data from metaphysics, considering that we don't really have a good place to write "integration" tests for this import script.